### PR TITLE
Typo fix + status syntax fix + WebDAV Update

### PR DESCRIPTION
**Pull request by Henrocker**
Typo fix + status sysntax fix

### CODE REVIEW

File: content/blog/caddy_and_owncloud.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ date: 2016-03-16 12:00:00+00:00
 
 In this post, I'll walk you through how to set up ownCloud with Caddy for a secure, personal cloud service. I wrote this guide while configuring on Ubuntu 14.
 
-**Update: Caddy's current release doesn't support WebDav yet, resulting in the desktop/mobile clients not working. Support for this is already included in the development branch, so you can either wait a until the next release, or compile Caddy yourself.**
+**Update: Caddy now supports WebDAV.**
```
Comment (mholt) [inline content/blog/caddy_and_owncloud.md]: We can take this line out entirely. ;)

File: content/blog/caddy_and_owncloud.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ I made the following Caddyfile together with *mholt* and *dprandzioch*. The conf
         # .htacces / data / config / ... shouldn't be accessible from outside
         rewrite {
             r  ^/(?:\.htaccess|data|config|db_structure\.xml|README)
-            status 403
+            to status 403
```
Comment (mholt) [inline content/blog/caddy_and_owncloud.md]: I think this now requires a separate directive entirely, the `status` middleware: https://caddyserver.com/docs/status

Comment (Henrocker) [inline content/blog/caddy_and_owncloud.md]: I've got it like this in production on my server and it works. In Administrator panel, it says all checks passed, which means htaccess, etc. are not accessible, thus the parameter can be expressed like above :)

File: content/blog/caddy_and_owncloud.md (hunk 1 above)
Comment (Henrocker) [inline content/blog/caddy_and_owncloud.md]: You are right, but I thought about the people revisiting the page after a while and seeing that the WebDav annotation is gone, that they might be thinking the feature is still not available for them, while the opposite is true. People might see the updated annotation and now can be sure WebDav works. Though I think in the near future we can delete the disclaimer.

File: content/blog/caddy_and_owncloud.md (hunk 2 above)
Comment (mholt) [inline content/blog/caddy_and_owncloud.md]: Hmmm... I still think this is not correct. Check the docs for rewrite. /cc @DenBeke

Comment (DenBeke) [inline content/blog/caddy_and_owncloud.md]: @Henrocker this used to work but the `rewrite` has changed as @mholt pointed out already.
What version of Caddy are you running?

Comment (Henrocker) [inline content/blog/caddy_and_owncloud.md]: I recompiled caddy from sources today. And the output from `./caddy -version` is: `Caddy (untracked dev build)`. As I stated, for me this works, becauseif I remove this line from my CaddyFile in my ownCloud section, the disclaimer appears again at the Administration panel.

Comment (mholt) [inline content/blog/caddy_and_owncloud.md]: That's weird. Build it with `build.bash` to ensure the proper version information is recorded and let us know what it says.

Comment (Henrocker) [inline content/blog/caddy_and_owncloud.md]: I've rebuilt it with `./build.bash` in the caddy directive but still, output is: `Caddy (untracked dev build)`

Comment (mholt) [inline content/blog/caddy_and_owncloud.md]: Then you are not running the built version...

Comment (Henrocker) [inline content/blog/caddy_and_owncloud.md]: How can I then?

Comment (Henrocker) [inline content/blog/caddy_and_owncloud.md]: I've got Caddy to recompile with the parameter name after the /build.bash command.

Output: Caddy 0.9.5 (+dc6c986 Mi Feb 01 17:27:47 UTC 2017)

Comment (mholt) [inline content/blog/caddy_and_owncloud.md]: There ya go! Does your Caddyfile work with that binary then? I would be surprised if it does.

**Pull request closed without merging**